### PR TITLE
AOB-1365 - AOB-1398 - Onboarder 4.0

### PR DESCRIPTION
**Pull request by CaillaudPA**
**Description**

- [AOB-1365](https://akeneo.atlassian.net/browse/AOB-1365) Update front-end command install
- [AOB-1398](https://akeneo.atlassian.net/browse/AOB-1398) Add documentation concerning the environment variables necessary for the proper functioning of the onboarder 

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Comment by nestorAkeneo** [comment]
:heart_eyes: Thanks for taking the time to contribute! You're awesome! :heart_eyes:

In order for your contribution to benefit all other contributors and users, we need your permission.

For that, it is enough that you read this document, and that you sign it, if it suits you.

[Contributor Agreement](http://www.akeneo.com/contributor-license-agreement/)

File: onboarder/installation/index.rst [hunk 1]
```diff
@@ -47,11 +47,35 @@ Build the UI
...
 .. code-block:: bash
 
     rm -rf var/cache/*
     bin/console cache:warmup --env=prod
-    bin/console pim:installer:assets --clean --env=prod && yarn run webpack
+    make front
+
+Else,
+
+For > 4.0.0 & <= 4.1.13 :
```
Comment (ValentinMerlet) [inline onboarder/installation/index.rst:59]: Unnecessary as long as you are targeting `4.0`

File: onboarder/installation/index.rst [hunk 2]
```diff
@@ -47,11 +47,35 @@ Build the UI
...
+.. code-block:: bash
+
+    rm -rf var/cache/*
+    bin/console cache:warmup --env=prod
+    bin/console pim:installer:assets --clean --env=prod
+    rm -rf public/dist
+    yarn run webpack
+
+For > 5.0.0 :
```
Comment (ValentinMerlet) [inline onboarder/installation/index.rst:68]: Add this part on a dedicated PR targeting the branch `5.0`

File: onboarder/installation/index.rst [hunk 3]
```diff
@@ -47,11 +47,35 @@ Build the UI
 
 Clear the Symfony cache and execute the following command to build the UI:
 
+If you have access to 'make' commands use :
 .. code-block:: bash
 
     rm -rf var/cache/*
     bin/console cache:warmup --env=prod
-    bin/console pim:installer:assets --clean --env=prod && yarn run webpack
+    make front
```
Comment (ValentinMerlet) [inline onboarder/installation/index.rst:55]: Imo, it's better to specify the commands they have to launch instead as Pierrick have suggested in your previous PR

File: onboarder/installation/define_environment_variables/dot_env_component.rst [hunk 4]
```diff
@@ -7,3 +7,13 @@ into the PIM as it's a standard Symfony framework component. It is also used by
 
 Edit the ``.env`` file at your project root directory and add the environment variables values provided on your
 Onboarder project page on the Partner Portal.
+
+.. code-block:: bash
```
Comment (ValentinMerlet) [inline onboarder/installation/define_environment_variables/dot_env_component.rst:11]: Why do you add those variables to the Onboarder documentation? They are related to the PIM.

**Review by ValentinMerlet — CHANGES_REQUESTED** [review]

File: onboarder/installation/index.rst (hunk 3 above)
Comment (CaillaudPA) [reply onboarder/installation/index.rst:55]: I have added both 

make and bin/console commands

File: onboarder/installation/define_environment_variables/dot_env_component.rst (hunk 4 above)
Comment (ValentinMerlet) [reply onboarder/installation/define_environment_variables/dot_env_component.rst:11]: @momoss Do you know why we have to tell the customer that they have to define those variables? If they are not defined, the PIM does not work at all

File: onboarder/installation/index.rst [hunk 5]
```diff
@@ -64,6 +65,20 @@ Make sure this file is shipped to the server which is hosting your PIM.
 This file must be accessible (read rights) by the system user that runs the pim (example: www-data).
 
 
+Check your .env file
```
Comment (PierrickMartos) [inline onboarder/installation/index.rst:68]:
```suggestion
Check your .env file, it must contains at least those variables
```

**Review by PierrickMartos — APPROVED** [review]

**Review by ValentinMerlet — APPROVED** [review]

**Comment by PierrickMartos** [comment]
@CaillaudPA faut que tu signes la CLA pour pouvoir merge : https://www.akeneo.com/contributor-license-agreement/

**Pull request merged**